InputTime/index.jsx

```diff
@@ -16,7 +16,6 @@ const renderAmPm = ({
 }) => {
   const style = calculateStyles({
     default: {
-      display: 'inline-flex',
```

Comment:
I think this makes sense, this was assuming it would be placed in a parent with `display: flex`